wpeverest/everest-forms
# Feature - Divider field added

### All Submissions:






### Changes proposed in this Pull Request:



Divider field added to Everest Forms Pro.

### How to test the changes in this Pull Request:

1. Create a form and Drag divider field from sidebar.
2. Change the divider type on field setting.
3. Verify the Changes in the UI, add some entries and view entries.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Divider field added
